roles/1-prep/tasks/raspberry_pi.yml

```
@@ -22,25 +22,32 @@
   when: rtc_id is defined and rtc_id != "none" and is_ubuntu    # CLARIF: Ubuntu runs increasingly well on RPi hardware, starting in 2020 especially
 
 
-- name: 'Install packages: fake-hwclock, dphys-swapfile'
+- name: 'Install package: fake-hwclock'
   package:
     name:
       - fake-hwclock      # 2021-03-15: Missing on Ubuntu etc.  RasPiOS installs this regardless -- to save/restore system clock on machines w/o working RTC (above).
-      - dphys-swapfile    # 2021-03-15: Missing on Ubuntu etc.  RasPiOS installs this regardless -- to autogenerate and use a swap file (below).
     state: present
 
-- name: Increase swap file size (to CONF_SWAPSIZE={{ pi_swap_file_size }} in /etc/dphys-swapfile) as kalite pip download fails
-  lineinfile:
-    path: "{{ etc_path }}/dphys-swapfile"
-    regexp: "^CONF_SWAPSIZE"
-    line: "CONF_SWAPSIZE={{ pi_swap_file_size }}"
-
-- name: Restart swap service "dphys-swapfile"
-  #command: /etc/init.d/dphys-swapfile restart
-  systemd:    # Had been...a rare/legacy service that was NOT systemd
-    name: dphys-swapfile
-    state: restarted
-
+- name: Install and configure dphys-swapfile
+  when: os_ver != "raspbian-13"
+  block:
+    - name: 'Install package: dphys-swapfile'
+      package:
+        name:
+          - dphys-swapfile    # 2021-03-15: Missing on Ubuntu etc.  RasPiOS uses zram now -- to autogenerate and use a swap file (below).
```

Optionally... combined onto 1 line:

```suggestion
        name: dphys-swapfile    # 2021-03-15: Missing on Ubuntu etc.  RasPiOS uses zram now -- to autogenerate and use a swap file (below).
```

---

I kept the same form as was present, the same could be said for fake-hwclock